near-rust-allocator-proxy/Cargo.toml

```diff
@@ -1,7 +1,14 @@
 [package]
 name = "near-rust-allocator-proxy"
-version = "0.2.6"
+version = "0.2.7"
 authors = ["Piotr Mikulski <piotr@near.org>"]
```

Comment:
@nearmax what is our policy regarding authors?

Reply:
I changed authors to be
```
authors = ["Near Inc <hello@nearprotocol.com>", "Piotr Mikulski <piotr@near.org>"]
```

Reply:
@nearmax @bowenwang1996  Is there anything else stopping us from publishing this crate?

Reply:
I don't think so. Please go ahead and publish the crate

Reply:
This should be fine.